.github/scripts/build-image.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# build and push docker image
+#
+
+set -o errexit
+set -o pipefail
+
+HELM_VERSIONS="$(grep 'ENV HELM2_VERSION' Dockerfile | sed -e 's/ENV HELM2_VERSION v//') $(grep 'ENV HELM_VERSION' Dockerfile | sed -e 's/ENV HELM_VERSION v//')"
+GCLOUD_SDK_VERSION="$(grep google/cloud-sdk Dockerfile | sed -e 's#FROM\ google/cloud-sdk:##'  -e 's#-alpine##')"
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+IS_MASTER="false"
+if [ "${GITHUB_REF}" == 'refs/heads/master' ]; then  IS_MASTER=true; fi
```

Comment:
Usually this would be done on a actions level, not within shell scripts.
Reference: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
Yes, of course. Later :-)
The main approach here is just to get it running like in CircleCI before